integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/mock/DistroXUpgradeTests.java

```diff
@@ -50,6 +50,10 @@ public void testDistroXUpgrade(TestContext testContext) {
                 .given(DistroXUpgradeTestDto.class)
                 .withRuntime(targetRuntimeVersion)
                 .given(distroXName, DistroXTestDto.class)
+                .then((tc, entity, client) ->  {
```

Comment:
Question: have you tried to set the CM version via the CM upgrade command? I mean during upgrade CM upgrade API command is called, it could then store the target CM version.

Reply:
the current logic in CB core doesn't make this possible as the version is checked after the package update. I don't want to change the code logic because of the test

Reply:
you are right.